content/en/getting_started/synthetics/api_test.md

```diff
@@ -22,60 +25,79 @@ further_reading:
-    - Choose from the **Managed Locations** to run your test from.
-    - Click the **Test URL** button.
+    - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, a test web application.
+    - You can select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - You can set environment tags such as `env:prod` and `app:shopist` and additional tags on your test. Tags allow you to stay organized and quickly find test you're interested in on the homepage.
+    - Click **Test URL**.
 
 {{< img src="getting_started/synthetics/api-test-config.png" alt="API test configuration"  style="width:60%;">}}
 
-#### Define your alert conditions
+#### Define assertions
+
+Datadog automatically populates basic assertions about your endpoint's response. Assertions define the alert condition and are customizable. 
 
-After clicking Test URL, basic assertions based on your endpoint's response are automatically populated. Assertions define the alert condition and can be customized. In this example, three default assertions are populated when testing the URL:
+In this example, three default assertions populate after testing the URL:
 
 {{< img src="getting_started/synthetics/assertions-example.png" alt="Browser test failure"  style="width:90%;">}}
 
-These assertions define the alert condition and can be customized. To add a custom assertion, directly click on the response preview or click the New Assertion button to add an assertion manually (e.g., body contains Shop.ist)
+To add a custom assertion, you can click anywhere in the response preview or click **New Assertion**.
 
 {{< img src="getting_started/synthetics/api-test-configuration.gif" alt="Browser test failure"  style="width:90%;">}}
 
-You can set up an alerting condition in order to only be alerted if your endpoint goes down for three minutes, on two different locations:
+#### Select locations 
+
+Select some **Managed Locations** or **Private Locations** to run your test from.
+
+#### Specify test frequency
+
+Select the interval of time you want to run your test.
+
+#### Define alert conditions
+
+You can specify the number of failures per location before the location is considered failed:
 
 ```text
-An alert is triggered if your test fails for 3 minutes from any 2 of 13 locations
+Retry x time before location is marked as failed
 ```
 
-You can also decide after how many failures you want a location to be considered as fail. Retries will be run immediately after the location fails. You can configure the test using the option below:
+After a location fails, retries immediately run.
+
+**Note**: By default, the wait time for a failed test to retry is 300ms. This interval can be configured via the [Synthetics API][3].
```

Comment:
```suggestion
**Note**: By default, the wait time for a failed test to retry is 300ms. You can confiture with the [Synthetics API][3].
```

Reply:
@kayayarai do you mean "configure with?"

Reply:
lol yes, setting configuration parameters, not french jam. And a better sentence would start: "You can configure the interval with"